sentry_sdk/integrations/starlette.py

```diff
@@ -423,6 +423,7 @@ async def __call__(self, scope, receive, send):
         hub = Hub.current
         integration = hub.get_integration(StarletteIntegration)
         if integration is None:
+            await self.app(scope, receive, send)
```

Comment:
Thanks for this -- v1.8.0 is ~unusable with POST requests Starlette without this change